utils/src/treev2/quickPickWizard/ContextValueQuickPickStep.ts

```diff
@@ -49,6 +23,8 @@ export class ContextValueQuickPickStep<TNode extends ContextValueFilterableTreeN
             (Array.isArray(excludeOption) ? excludeOption : [excludeOption]) :
             [];
 
+        node = isBox(node) ? node.unwrap() : node;

```

Comment:
Actually this might not be what we want to do (not yet). RG is going to wrap every node in the tree; VSCode will be calling things like `getChildren()` using those wrapped nodes--not the item inside the box.

We should change to do the unwrapping only at the end of the experience.

Reply:
Relatedly: it will be the responsibility of the wrapper object that RG is using to reflect the context values of the wrapped object (plus maybe some additional ones). That way, the filters will still "see" the context values of the item inside the box.

Reply:
> (plus maybe some additional ones)

If we want to avoid unwrapping, we'll have to reflect these as well:

* `resourceType` for `GroupItem`
* `resource` for `AppResourceItem`